packages/core/src/pipeline/config/triggers/ExecutionOptionsPageContent.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import { ExecutionOptionsPageContent } from './ExecutionOptionsPageContent';
+import type { IPipeline } from '../../../domain';
+
+describe('Execution Options Page Content', () => {
+  describe('Max Concurrent Options', () => {
+    let pipeline: IPipeline;
+    const setPipeline = (overrides: any = {}) => {
+      pipeline = {
+        application: 'test',
+        id: 'test1',
+        keepWaitingPipelines: false,
+        limitConcurrent: false,
+        maxConcurrentExecutions: 0,
```

Comment:
@guido9j I did have a question on this - right now it's a required value based on the type definition, should it be optional?